src/query/pipeline/transforms/src/processors/transforms/sort/rows/simple.rs

```diff
@@ -81,7 +81,7 @@ where
     T::Scalar: Ord,
 {
     fn eq(&self, other: &Self) -> bool {
-        self == other
+        self.inner == other.inner && self.desc == other.desc
```

Comment:
Should desc be ignored or is it inconsistent with cmp behavior.

Reply:
the descs are consistent.